facade/servicetemplate.go

```diff
@@ -24,75 +24,88 @@ import (
 	"github.com/control-center/serviced/domain/service"
 	"github.com/control-center/serviced/domain/servicedefinition"
 	"github.com/control-center/serviced/domain/servicetemplate"
+	"github.com/control-center/serviced/audit"
+	"github.com/Sirupsen/logrus"
 )
 
 var getDockerClient = func() (*dockerclient.Client, error) { return dockerclient.NewClient("unix:///var/run/docker.sock") }
 
 //AddServiceTemplate  adds a service template to the system. Returns the id of the template added
 func (f *Facade) AddServiceTemplate(ctx datastore.Context, serviceTemplate servicetemplate.ServiceTemplate, reloadLogstashConfig bool) (string, error) {
 	defer ctx.Metrics().Stop(ctx.Metrics().Start("Facade.AddServiceTemplate"))
+	alog := f.auditLogger.Message(ctx, "Adding Service Template").
```

Comment:
I don't believe the `ID` is set at this point in time - see line 43 below. I think this message needs to be moved to line 44